client/components/tinymce/plugins/wpcom-view/views/simple-payments/style.scss

```diff
@@ -82,116 +82,48 @@
 	flex: 0 0 auto;
 }
 
-.wpview-type-simple-payments__pay-paypal-button {
-	width: 150px;
-	position: relative;
-	top: 50%;
-	transform: translateY( -50% );
-	background-position: 0px 50%;
-	box-sizing: border-box;
-	height: 30px;
-	max-width: 200px;
-	min-height: 22px;
-	min-width: 80px;
-	text-align: center;
-	white-space: nowrap;
-	width: 148px;
-	perspective-origin: 74px 15px;
-	transform-origin: 74px 15px;
-	user-select: none;
-	background: rgba(0, 0, 0, 0) none repeat scroll 0px 50% / auto padding-box border-box;
-	font: normal normal 500 normal 16px / normal "Helvetica Neue", Helvetica, Arial, sans-serif;
-	padding: 2px;
-}
-
 .wpview-type-simple-payments__pay-paypal-button-content {
-	bottom: 0px;
 	box-sizing: border-box;
-	cursor: pointer;
 	height: 24px;
-	left: 0px;
-	max-height: 24px;
 	position: relative;
-	right: 0px;
 	text-align: center;
-	text-shadow: rgb(204, 204, 204) 0px -1px 0px;
 	top: 50%;
 	transform: translateY( -50% );
-	white-space: nowrap;
 	width: 144px;
-	perspective-origin: 72px 12px;
-	transform-origin: 72px 12px;
-	background: rgb(238, 238, 238) none repeat scroll 0% 0%  auto padding-box border-box;
-	border-radius: 4px 4px 4px 4px;
-	font: normal normal 500 normal 10px / normal "Helvetica Neue", Helvetica, Arial, sans-serif;
 	padding: 4px 8px;
 }
 
 .wpview-type-simple-payments__pay-paypal-button-content:before {
```

Comment:
Could this style be merged into the `.wpview-type-simple-payments__pay-paypal-button-content` element, specifying the gradient background and rounded corners for the button? The element would get 2px higher (24px to 26px), but that doesn't seem to hurt anything.